includes/learnopengl/model.h

```diff
@@ -35,6 +37,55 @@ class Model
     // constructor, expects a filepath to a 3D model.
     Model(string const &path, bool gamma = false) : gammaCorrection(gamma)
     {
+        defaultDiffuse = 0;
+        defaultSpecular = 0;
+        defaultNormal = 0;
+        defaultOther = 0;
```

Comment:
You don't want to set any of these to 0 prior to checking if they were.
Also, they're static member. Which mean that the same texture is shared to all members of class Model.
It start at 0 which is invalid for texture until it's created here. You get 1 default texture for each type.

Reply:
Whoops, that was left from testing something...